src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/App.razor

```diff
@@ -1,4 +1,5 @@
-﻿<CascadingValue Value="Visual.Fluent" IsFixed="true">
+﻿<PageTitle>Bit's Native Blazor UI Components</PageTitle>
```

Comment:
Just to let you know that from now, we can use <PageTitle> in our razor pages so we can use the customized titles/meta tags for each page. @msynk @maryamhdr